litex/build/sim/core/modules.c

```diff
@@ -50,7 +50,7 @@ int litex_sim_load_ext_modules(struct ext_module_list_s **mlist)
   tinydir_file file;
   dylib_ref lib;
   int (*litex_sim_ext_module_init)(int (*reg)(struct ext_module_s *));
-  char name[100];
+  char name[300];
```

Comment:
This one is not so pretty but...
_TINYDIR_FILENAME_MAX is 256 so 300 is an easy fix.